tools/opcode_metadata/generate_rs_opcode_metadata.py

```
@@ -0,0 +1,364 @@
+#!/usr/bin/env python
+from __future__ import annotations
+
+import collections
```

Flake8 (7.3.0)

[error] 'collections' imported but unused

(F401)